frontend/device/kindle/device.lua

```diff
@@ -66,13 +66,22 @@ function Kindle:supportsScreensaver()
     end
 end
 
-function Kindle:setTime(hour, min)
+function Kindle:setTime(year, month, day, hour, min, sec)
```

Comment:
I'd say that it should be renamed to setDate with the new functionality.

Possibly with a convenience front like
```lua
function setTime(hour, min, sec)
  self:setDate(nil, nil, nil, hour, min, sec)
end
```